edgePy/edgePy.py

```diff
@@ -0,0 +1,37 @@
+
+import argparse
+from edgePy.io import DataImporter, GroupImporter
+
+
+def parse_arguments(parser=None):
+    if not parser:
+        parser = argparse.ArgumentParser()
+
+    parser.add_argument("--count_file", help="name of the file to tweet")
+    parser.add_argument("--groups_file", help="Where to put the combined files")
+    args = parser.parse_args()
+
+    return args
+
+
+class edgePy(object):
```

Comment:
Limit the script to have only the wrapper for argument parsing.

Move all other code to internal library.

This is commonly done to abstract away the CLI interface with the actual interactive API. It makes unit testing a ton easier too because you can test the CLI interface and the classes/functions behind it seperately!